packages/at_client/lib/src/client/at_client_impl.dart

```diff
@@ -261,11 +261,11 @@ class AtClientImpl implements AtClient, AtSignChangeListener {
 
   /// Does nothing unless a telemetry service has been injected
   void _cascadeSetTelemetryService() {
-    if (telemetry != null) {
-      _encryptionService?.telemetry = telemetry;
-      _localSecondary?.telemetry = telemetry;
-      _remoteSecondary?.telemetry = telemetry;
-    }
+    // if (telemetry != null) {
+    //   _encryptionService?.telemetry = telemetry;
+    //   _localSecondary?.telemetry = telemetry;
+    //   _remoteSecondary?.telemetry = telemetry;
+    // }
```

Comment:
the above objects were not using telemetry so this is cleanup but an aide-memoire as to intent